src/aiortc/rtcpeerconnection.py

```diff
@@ -1078,7 +1078,7 @@ def __createSctpTransport(self) -> None:
         self.__sctp._bundled = False
         self.__sctp.mid = None
 
-        @self.__sctp.on("datachannel")  # type: ignore
+        @self.__sctp.on("datachannel")
```

Comment:
Now `.on` is correctly type-inferred due to 
> Introduce overloads for ee.on

[in `pyee==13.0.0`](https://github.com/jfhbrook/pyee/releases/tag/v13.0.0).